CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This file is used to track changes made in recent versions of the Sensu
 cookbook. Please see HISTORY.md for changes from older versions of this project.
 
 ## [Unreleased]
+* Enable gpg check for all linux repo installs using a key downloaded over HTTPS. Download windows MSI over HTTPS. #578 (@mike-stewart)
```

Comment:
I can fix it up post acceptance but you should use a `### Security` header for security changes.

We follow the same changelog guidelines as [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) and if you are curious on how we version security changes you can read about it [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_VERSION.md)